packages/core/src/components/Translate/index.tsx

```diff
@@ -4,29 +4,25 @@ import { TranslateParams, TranslateProps, TranslateOptions } from '../../types';
 
 export type Props = TranslateProps;
 
-/** Translate a phrase with a key, informational context, and dynamic params. */
+/** Translate a phrase with a key and dynamic params. */
 export default class Translate extends React.PureComponent<Props> {
   static defaultProps = {
     html: false,
   };
 
-  static phrase(
-    phrase: string,
-    params?: TranslateParams | null,
-    options?: string | TranslateOptions,
-  ): string {
-    return Core.translate(phrase, params, options);
+  static phrase(key: string, phrase: string, params?: TranslateParams): string {
```

Comment:
very breaking! I think OK if we ship the code mod and directions on how to update

Reply:
Yup, this is for v3.